packages/react-native-editor/__device-tests__/gutenberg-editor-lists-@canary.test.js

```diff
@@ -8,27 +8,28 @@ import testData from './helpers/test-data';
 describe( 'Gutenberg Editor tests for List block', () => {
 	it( 'should be able to add a new List block', async () => {
 		await editorPage.addNewBlock( blockNames.list );
-		const listBlockElement = await editorPage.getBlockAtPosition(
-			blockNames.list
-		);
-		// Click List block on Android to force EditText focus
-		if ( isAndroid() ) {
-			await listBlockElement.click();
-		}
+		let listBlockElement = await editorPage.getListBlock( {
+			isEmptyBlock: true,
+		} );
 
-		// Send the first list item text.
-		await editorPage.sendTextToListBlock(
+		await editorPage.typeTextToTextBlock(
 			listBlockElement,
-			testData.listItem1
+			testData.listItem1,
+			false
 		);
 
+		listBlockElement = await editorPage.getListBlock();
+
 		// Send an Enter.
-		await editorPage.sendTextToListBlock( listBlockElement, '\n' );
+		await editorPage.typeTextToTextBlock( listBlockElement, '\n', false );
+
+		listBlockElement = await editorPage.getListBlock();
```

Comment:
If the block was already retrieved in line `21, do we need need to query it again?

Reply:
Retested this and looks like it is indeed not needed, I've removed it in [this commit](https://github.com/WordPress/gutenberg/pull/40670/commits/46273a83845071272a8950c58365a30608581a1e#diff-53684945abd1075e62d91961d64621a1f8a9c5a3189330c3d5a6b2159f0513cb)

Reply:
Great, thanks for checking it 🙇 .